modules/flight_interface/flight_interface_worker.py

```diff
@@ -45,4 +49,10 @@ def flight_interface_worker(
         if not result:
             continue
 
+        try:
```

Comment:
Add a comment to explain why this is being done. Could also add a comment above for the magic number 1.